config/_default/security.toml

```diff
@@ -3,7 +3,7 @@
 
   [exec]
     allow = ['^go$']
-    osEnv = ['^PATH$']
+    osEnv = ['^PATH$|TMP']
 
```

Comment:
I suggest that we remove `allow` and `osEnv` config and just rely on the default settings for those, which has been improved a lot since the above was added.

Reply:
Reverted to the default settings, as requested.